packages/synapse-react-client/src/components/ChallengeDataDownload/Renderers.tsx

```diff
@@ -34,10 +38,19 @@ export function SizeRenderer<T extends EntityIdAndVersionNumber>(
 export function DownloadRenderer<T extends EntityIdAndVersionNumber>(
   props: CellRendererProps<T>,
 ) {
+  const { data: entity } = useGetEntity<FileEntity>(
+    props.rowData.entityId,
+    props.rowData.versionNumber,
+  )
+  if (!entity) {
+    return <></>
+  }
   return (
     <DirectDownload
+      fileHandleId={entity?.dataFileHandleId}
       associatedObjectId={props.rowData.entityId}
+      associatedObjectType={FileHandleAssociateType.FileEntity}
```

Comment:
Changed `DirectDownload` to no longer accept an entity ID, you must now provide the full FileHandleAssociation tuple.

So this is one of a few places where we must now fetch the FileEntity beforehand.